sdk/resourcemanager/Azure.ResourceManager/CHANGELOG.md

```diff
@@ -1,12 +1,9 @@
 # Release History
 
-## 1.13.0-beta.1 (Unreleased)
+## 1.13.0 (2024-09-10)
 
 ### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+- Added `JsonModelWriteCore` to serialization file so that derived models could call it without do the serialization of base properties itself.
```

Comment:
```suggestion
- Added `JsonModelWriteCore` to serialization file so that derived models could call it without doing the serialization of base properties themself.
```